src/runtime/internal/utils.ts

```diff
@@ -90,13 +90,13 @@ export function once(fn) {
 	}
 }
 
-const is_client = typeof window !== 'undefined';
+export const is_client = typeof window !== 'undefined';
 
 export let now: () => number = is_client
 	? () => window.performance.now()
 	: () => Date.now();
 
-export let raf = is_client ? requestAnimationFrame : noop;
+export let raf = is_client ? cb => requestAnimationFrame(cb) : noop;
```

Comment:
I'm confused about why I had to add this to get the tests to even start. I was getting a weird error about `requestAnimationFrame` not being defined. (I also couldn't use `export let raf = is_client ? window.requestAnimationFrame : noop;` because that broke treeshaking.)